src/raqm.c

```diff
@@ -1121,40 +1216,85 @@ _raqm_itemize (raqm_t *rq)
     ok = false;
     goto done;
   }
+#endif
 
   if (rq->base_dir == RAQM_DIRECTION_RTL)
-    par_type = FRIBIDI_PAR_RTL;
+    par_type = RAQM_PAR_RTL;
   else if (rq->base_dir == RAQM_DIRECTION_LTR)
-    par_type = FRIBIDI_PAR_LTR;
+    par_type = RAQM_PAR_LTR;
 
   if (rq->base_dir == RAQM_DIRECTION_TTB)
   {
     /* Treat every thing as LTR in vertical text */
     max_level = 1;
-    memset (types, FRIBIDI_TYPE_LTR, rq->text_len);
+#ifndef RAQM_SHEENBIDI
+    memset (types, RAQM_TYPE_LTR, rq->text_len);
```

Comment:
Why is this behind an ifdef, if the direction is vertical, we skip vidi and set all types and levels to 0, this should equally happen regardless of the bid implementation used.

Reply:
fixed in f373592feafe1ac89fcd161ed9299ff70be35dfa